api/HTMLInputElement.json

```
@@ -1286,6 +1286,63 @@
           }
         }
       },
+      "setRangeText": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/setRangeText",
+          "support": {
+            "chrome": {
+              "version_added": true
```

```suggestion
              "version_added": "24"
```
Per https://trac.webkit.org/changeset/131969/webkit and https://trac.webkit.org/browser/webkit/tags/Safari-537.17/Source/WebCore/html/HTMLInputElement.idl#L82, this was in WebKit 537.17, and so Chrome 24.